torch/_torch_docs.py

```diff
@@ -8810,6 +8810,33 @@ def merge_dicts(*dicts):
     tensor([ 4,  5,  8])
 """.format(**common_args))
 
+add_docstr(torch.take_along_dim,
+           r"""
+take_along_dim(input, indices, dim, *, out=None) -> Tensor
+
+Returns values from the :attr:`input` at the given 1-d indices along the given dim.
+
+.. note::
+    This function is similar to NumPy's `take_along_axis`.
+    See also :func:`torch.gather`.
+
+Args:
+    {input}
+    indices (LongTensor): the indices into tensor
+    dim (int): dim to take 1d slices along.
+
+Keyword args:
+    {out}
+
+Example::
+
+    >>> a = torch.tensor([[10, 30, 20], [60, 40, 50]])
+    >>> sorted_idx = torch.tensor([[0, 2, 1], [1, 2, 0]])
+    >>> torch.take_along_dim(a, sorted_idx, dim=1)
```

Comment:
Would you update the docs for argmin and argmax, too? I think they should be paired with this function.

See https://github.com/pytorch/pytorch/pull/49684#discussion_r546834844 and the NumPy documentation.